src/components/ebay-menu/index.js

```diff
@@ -94,7 +94,9 @@ function getInitialState(input) {
         priority: input.priority,
         expanded: false,
         items,
-        checked: checkedItems
+        checked: checkedItems,
+        customLabel: input.label,
```

Comment:
I would keep consistency between both the variable passed in and the state/data variable defined. 
So either change it to be either:
checked: checked,
label: input.label

OR
checkedItems: checkedItems,
customLabel: input.customLabel

That way it's easy to see where the variables come from

Reply:
@agliga I think the label is the only code here. We should leave the `checked: checkedItems` part untouched to prevent risk of regression/breaking change.